climpred/smoothing.py

```diff
@@ -132,145 +136,110 @@ def check_lon_lat_present(da):
-    return ds_out
-
-
-@is_xarray(0)
-def temporal_smoothing(ds, smooth_kws=None, how='mean', rename_dim=True):
+def temporal_smoothing(
+    ds, tsmooth_kws=None, how='mean', rename_dim=True, d_lon_lat_kws=None
+):
     """Apply temporal smoothing by creating rolling smooth-timestep means.
 
     Reference:
     * Goddard, L., A. Kumar, A. Solomon, D. Smith, G. Boer, P.
      Gonzalez, V. Kharin, et al. “A Verification Framework for
      Interannual - to - Decadal Predictions Experiments.” Climate
      Dynamics 40, no. 1–2 (January 1, 2013): 245–72.
-     https: // doi.org / 10 / f4jjvf.
+     https://doi.org/10/f4jjvf.
 
     Args:
         ds(xr.object): input.
-        smooth_kws(dict): length of smoothing of timesteps.
-                          Defaults to {'time': 4} (see Goddard et al. 2013).
+        tsmooth_kws(dict): length of smoothing of timesteps.
+            Defaults to {'time': 4} (see Goddard et al. 2013).
         how(str): aggregation type for smoothing. default: 'mean'
         rename_dim(bool): Whether labels should be changed to
-                          `'1-(smooth-1)', '...', ...`. default: True.
+            `'1-(smooth-1)', '...', ...`. default: True.
+        d_lon_lat_kws (None): leads nowhere but consistent with
+            `spatial_smoothing_xesmf`.
 
     Returns:
         ds_smoothed(xr.object): input with `smooth` timesteps less
-        and labeling '1-(smooth-1)', '...', ... .
+            and labeling '1-(smooth-1)', '...', ... .
 
     """
     # unpack dict
-    if not isinstance(smooth_kws, dict):
-        raise ValueError('Please provide smooth_kws as dict, found ', type(smooth_kws))
-    if not ('time' in smooth_kws or 'lead' in smooth_kws):
+    if not isinstance(tsmooth_kws, dict):
+        raise ValueError(
+            'Please provide tsmooth_kws as dict, found ', type(tsmooth_kws)
+        )
+    if not ('time' in tsmooth_kws or 'lead' in tsmooth_kws):
         raise ValueError(
-            'smooth_kws doesnt contain a time dimension \
+            'tsmooth_kws doesnt contain a time dimension \
             (either "lead" or "time").',
-            smooth_kws,
+            tsmooth_kws,
         )
-    smooth = list(smooth_kws.values())[0]
-    dim = list(smooth_kws.keys())[0]
+    smooth = list(tsmooth_kws.values())[0]
+    dim = list(tsmooth_kws.keys())[0]
     # fix to smooth either lead or time depending
     time_dims = ['time', 'lead']
     if dim not in ds.dims:
         time_dims.remove(dim)
         dim = time_dims[0]
-        smooth_kws = {dim: smooth}
+        tsmooth_kws = {dim: smooth}
     # aggreate based on how
-    ds_smoothed = getattr(ds.rolling(smooth_kws, center=False), how)()
+    ds_smoothed = getattr(ds.rolling(tsmooth_kws, center=False), how)()
     # remove first all-nans
     ds_smoothed = ds_smoothed.isel({dim: slice(smooth - 1, None)})
     ds_smoothed[dim] = ds.isel({dim: slice(None, -smooth + 1)})[dim]
     if rename_dim:
```

Comment:
Is this needed? It's defaulted to `True` but the only other place it's referenced is in the `.smooth()` method in classes and it's passed as `False` there.

Reply:
This is from the previous code. I use this default in the direct function testing. Not strictly needed anymore. True

Reply:
removed now